tests/Groups/GroupTest.php

```diff
@@ -76,4 +76,14 @@ public function testIterator()
 
         $this->assertEquals(['name', 'surname', 'address'], $keys);
     }
+
+    public function testInvalidInput()
+    {
+        $group = new Group();
+
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('The element "name" must implement');
```

Comment:
Hi. Thanks for this. Just one question: Seems like the error message is incomplete. `The element "name" must implement` should be `The element "name" must implement FormManager\InputInterface`.

Reply:
@oscarotero, fixed!